src/components/detailsAndImage.tsx

```diff
@@ -78,6 +87,19 @@ const titleCss = css`
   ${from.desktop} {
     font-size: 24px;
   }
+
+  ${sectionTitle === "Reader funding" &&
+  `
+    font-size: 24px;
+
+    ${from.tablet} {
```

Comment:
Does the font size need resizing to 24px at tablet and wide breakpoints if its set to 24px at the top?

Reply:
Ok, thanks, all signed off